src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Action.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import actionStyles from './action.scss';
+
+type Props = {
+    children?: string,
+    onClick: () => void,
+    afterAction?: () => void,
+};
+
+export default class Action extends React.PureComponent<Props> {
+    handleButtonClick = () => {
+        this.props.onClick();
+        if (this.props.afterAction) {
```

Comment:
The `onClick` is set by the user (developer) of this component. The `afterAction` however is used by the select component to close the overlay list when the action has been clicked.